src/main/java/com/launchacademy/reviews/seeders/ReviewSeeder.java

```diff
@@ -0,0 +1,21 @@
+package com.launchacademy.reviews.seeders;
+
+import com.launchacademy.reviews.services.ReviewService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReviewSeeder {
+  private ReviewService reviewService;
+  private SiteService siteService;
+
+  public ReviewSeeder(ReviewService reviewService, SiteService siteService) {
+    this.reviewService = reviewService;
+    this.siteService = siteService;
+  }
+
+  public void seed(){
+    if(reviewService.findAll().isEmpty()){
+      Site site = siteService.findByName("").get(0);
+    }
```

Comment:
After finding a site we can add some reviews to it

Reply:
Yes, our next story is to add a form component on each site show page to add reviews.  This story is just to seed and display reviews.